driver/driver.cpp

```diff
@@ -581,7 +587,16 @@ Routine Description:
             DebugPrint(("WdfRequestRetrieveInputBuffer failed %x\n", status));
             break;
         }
-        
+
+        ULONG resultLen;
+        status = WdfDeviceQueryProperty(hDevice, DevicePropertyHardwareID,
```

Comment:
this will try to get the whole hwid list (`REG_MULTI_SZ`)

should probably alloc here (non-paged), based on `REGSTR_VAL_MAX_HCID_LEN`
then set `devExt->hwid` to the first id (the main/device id by convention)

Reply:
I'm not so familiar with this. Could you make this change + defining somewhere REGSTR_VAL_MAX_HCID_LEN? (it must be accesible also in wrapper.cpp)

Reply:
sure